core/config/general_config.go

```diff
@@ -484,6 +486,37 @@ func (c *generalConfig) AppID() uuid.UUID {
 	return c.appID
 }
 
+func (c *generalConfig) AuditLoggerEnabled() bool {
+	return c.viper.GetBool(envvar.Name("AuditLoggerEnabled"))
+}
+
+func (c *generalConfig) AuditLoggerForwardToUrl() (models.URL, error) {
+	url, err := models.ParseURL(c.viper.GetString(envvar.Name("AuditLoggerForwardToUrl")))
+	if err != nil {
+		return models.URL{}, err
+	}
+	return *url, nil
+}
+
+func (c *generalConfig) AuditLoggerEnvironment() string {
+	if c.viper.GetBool(envvar.Name("Dev")) {
```

Comment:
Is there a `Dev()` method to call instead?
```suggestion
	if c.Dev() {
```